src/plus/webviews/graph/protocol.ts

```diff
@@ -61,6 +61,17 @@ export const enum GraphScrollMarkerTypes {
 	Upstream = 'upstream',
 }
 
+export const enum GraphMinimapMarkerTypes {
+	Selection = 'selection',
+	Head = 'head',
+	Highlights = 'highlights',
+	LocalBranches = 'localBranches',
+	RemoteBranches = 'remoteBranches',
+	Stashes = 'stashes',
+	Tags = 'tags',
+	Upstream = 'upstream',
+}
+
```

Comment:
I noticed this enum doesn't match the `GraphMinimapTypes` in `config.ts`. Should these match?

Reply:
HEAD and upstream are controlled by a single `head` setting in the configuration, but they are two different markers in the minimap. Similar setup with scroll markers.

Reply:
But I guess since some of these are always-on, we don't need to include them in the config. Let me take a look at it. We might be able to remove `Head`, `Selection`, and `Highlights` from `GraphMinimapTypes` and `GraphScrollMarkerTypes`.

Reply:
Let's keep the full list for now and we can trim down later if needed. 👍🏼